styleguide/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ama-style-guide",
-  "version": "0.0.8",
+  "version": "1.0.1",
```

Comment:
My only question is that I'm confused about this.  It looks like 1.0.1 was the release PRIOR to 1.0.0, which may have just been a mistake on megh's part.  we should update this to correspond with the sprint number now, correct?  Like if I'm going to cut a release on this today, I would name it v1.11.0, correct?

Reply:
Correct.